Github Issue Reader/ViewController.swift

```diff
@@ -126,11 +126,12 @@ class ViewController: UIViewController {
         viewModel.fetchIssues(for: organization, repo: repo) { [self] issues in
             print("Successfully received issues.")
             
-            for issue in self.viewModel.issues {
-                print(issue.title)
-            }
+//            for issue in self.viewModel.issues {
+//                print(issue.title)
```

Comment:
I try not to check in caveman debugging or development printage.  (And also try not to check in commented-out swaths of code)